### ISSUE TITLE
Issue with Angular 4 and AoT

### ISSUE THREAD

**Issue by DavidRouyer** [dominique-mueller/angular-notifier#12]
I'm using the Angular-CLI (v1.0.0), I have successfully integrated angular-notifier but I get an error when I try to run ng build --prod --aot:
```
ERROR in ng:///D:/Documents/GitHub/hopfab/backoffice/node_modules/angular-notifier/src/components/notifier-container.com
ponent.d.ts.NotifierContainerComponent.html (2,9): Property 'notification' does not exist on type 'NotifierContainerComp
onent'.
ng:///D:/Documents/GitHub/hopfab/backoffice/node_modules/angular-notifier/src/components/notifier-container.component.d.
ts.NotifierContainerComponent.html (2,9): Property 'notification' does not exist on type 'NotifierContainerComponent'.

ERROR in ng:///D:/Documents/GitHub/hopfab/backoffice/node_modules/angular-notifier/src/components/notifier-container.com
ponent.d.ts.NotifierContainerComponent.html (2,9): Property 'notification' does not exist on type 'NotifierContainerComp
onent'.
ng:///D:/Documents/GitHub/hopfab/backoffice/node_modules/angular-notifier/src/components/notifier-container.component.d.
ts.NotifierContainerComponent.html (2,9): Property 'notification' does not exist on type 'NotifierContainerComponent'.
```

**Comment by dominique-mueller** [comment]
Yep! I had the same issue a few days ago when upgrading an Angular application from 2 to 4.

Within the notification container, we're displaying the notification with a ngFor loop. For performance reasons, I've also added implemented the trackby functionality. Turns out, lots of examples out there are actually wrong - and so is the implementation within this library.

This situation has also been described in https://github.com/angular/angular/issues/13388, hopefully we can expect a simpler syntax proposed in https://github.com/angular/angular/issues/12969.

I'll fix it and publish a new release soon.
